pkgs/development/compilers/sagittarius-scheme/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, cmake
+, pkgconfig
+, libffi
+, boehmgc
+, openssl
+, zlib
+, odbc ? true
+, libiodbc
+}:
+
+let version = "0.9.6";
+    odbcDeps = if odbc then [ libiodbc ] else [];
+in stdenv.mkDerivation {
+  name = "sagittarius-scheme-${version}";
+  src = fetchurl {
+    url = "https://bitbucket.org/ktakashi/sagittarius-scheme/downloads/sagittarius-${version}.tar.gz";
+    sha256 = "03nvvvfd4gdlvq244zpnikxxajp6w8jj3ymw4bcq83x7zilb2imr";
+  };
+  preBuild = ''
+           export LD_LIBRARY_PATH="$(pwd)/build"
```

Comment:
Why is this required?

Reply:
Sorry, I will add a comment in the file, though the reason is we (Nix's cmake tooling) skip adding rpaths during build (`-DCMAKE_SKIP_BUILD_RPATH=ON`) and build of Sagittarius involves building a binary executable `sagittarius` (which is linked to its own `libsagittarius.so`) and is then required to be executed to process couple of Scheme files, and dies due to not being able to resolve `libsagittarius.so` library (in the absence of `LD_LIBRARY_PATH` or `rpath`).

Reply:
OK. Please keep indentation consistent at 2 spaces per level.